src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -4819,18 +4819,28 @@ QGISEXTERN QString loadStyle( const QString &uri, QString &errCause )
     return QString();
   }
 
+  QString geomColumnExpr;
```

Comment:
Candidate for `QgsExpression::fieldEqualityExpression()`?

Reply:
Hmm not sure about that - different providers have different quotedValue() implementations :-/

Reply:
Oh, I wasn't aware of that.
Might be nice to have the same implemented per provider in this case to avoid code duplication.